app/messaging/submit-claim.js

```diff
@@ -27,7 +28,9 @@ const submitClaim = async (message) => {
     const res = await updateByReference({ reference, claimed: true, updatedBy: 'admin' })
 
     const updateSuccess = isUpdateSuccessful(res)
+
     if (updateSuccess) {
+      await submitPaymentRequest(application.dataValues, message.sessionId)
```

Comment:
What do you think to attempting the payment request before the update to the database? I'm thinking about what happens when errors occur.

Currently the DB will have been updated to show it has been claimed but when an error happens during payment no payment will be made it isn't going to be possible to make a claim on an already claimed application. However, if the payment request is made then the DB could be updated. Obviously if the DB update fails there is potential for another payment request to be made.
Does there need to be an additional state added? Something like `payment-inprogress` that is set when the function is started and would it be possible to have the payment response update the state to `payment-complete`?